sites/main-site/src/content/blog/2024/seqera-containers-part-2.mdx

```diff
@@ -0,0 +1,646 @@
+---
+title: 'Migration from Biocontainers to Seqera Containers: Part 2'
+subtitle: "nf-core containers automation: how it'll all work behind the curtain"
+pubDate: 2024-10-02T09:20:00+01:00
+headerImage: https://images.unsplash.com/photo-1711109631679-c9e1094d3118
+headerImageAlt: Photo by Rafael Garcin on Unsplash
+authors:
+    - 'ewels'
+    - 'edmundmiller'
+label:
+    - 'modules'
+    - 'wave'
+    - 'seqera containers'
+embedHeaderImage: false
+---
+
+import creation_flow from '@assets/images/blog/seqera-containers-part-2/creation_flow.excalidraw.svg?raw';
+import renovate_flow from '@assets/images/blog/seqera-containers-part-2/renovate_flow.excalidraw.svg?raw';
+
+import { Image } from 'astro:assets';
+import { YouTube } from '@astro-community/astro-embed-youtube';
+
+# Introduction
+
+In nf-core, we've been excited to adopt [Wave](https://seqera.io/wave/) to automte software container builds, and have been
+looking for the right way to do it.
+With the announcement of [Seqera Containers](https://seqera.io/containers/) we felt it was the right time to put in
+the effort to migrate our containers to be built using Wave, using Seqera Containers to host the container images for our modules.
+You can read more about our motivation for this change in [Part 1 of this blog post](https://nf-co.re/blog/2024/seqera-containers-part-1).
+
+Here, in Part 2, we will dig into the technical details: how it all works behind the curtain.
+You don't need to know or understand any of this as an end-user of nf-core pipelines,
+or even as a contributor to nf-core modules, but we thought it would be interesting to share the details.
+It's mostly to serve as an architectural plan for the nf-core maintainers and infrastructure teams.
+
+:::tip{.fa-hourglass-clock title="Too Long; Didn't Read"}
+
+In a hurry? Here are the main points:
```

Comment:
```suggestion
```
no time for that 🙂

Reply:
![](https://miro.medium.com/v2/resize:fit:1000/format:webp/1*wuhil5dHn7k-RYG7JW3Uqw.gif)